src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int topLength;
+    private int bottomLength;
+    private int height;
+
+    public IsoscelesTrapezoid(int topLength, int bottomLength, int height, Color color) {
+        setFigure(FigureType.ISOSCELES_TRAPEZOID);
+        setTopLength(topLength);
+        setBottomLength(bottomLength);
+        setHeight(height);
+        setColor(color);
+    }
+
+    public IsoscelesTrapezoid() {
+        setFigure(FigureType.ISOSCELES_TRAPEZOID);
+        setTopLength(2);
+        setBottomLength(2);
+        setHeight(2);
+        setColor(Color.BLUE);
+    }
+
+    @Override
+    public double getCalculatedArea() {
+        return ((bottomLength * topLength) * height) / 0.5;
```

Comment:
The formula for calculating the area of an isosceles trapezoid is incorrect. It should be `((topLength + bottomLength) * height) / 2` instead of `((bottomLength * topLength) * height) / 0.5`. This is a critical error that needs to be fixed to correctly calculate the area of the trapezoid.